tests/mir-opt/copy-prop/move_projection.rs

```diff
@@ -15,6 +14,12 @@ struct Foo(u8);
 
 #[custom_mir(dialect = "runtime")]
 fn f(a: Foo) -> bool {
+    // CHECK-LABEL: fn f(
+    // CHECK-SAME: [[a:_.*]]: Foo)
+    // CHECK: bb0: {
+    // CHECK-NOT: _2 = copy [[a]];
+    // CHECK-NOT: _3 = move (_2.0: u8);
+    // CHECK: _3 = copy ([[a]].0: u8);
```

Comment:
Could you add the `CHECK: _0 = opaque(copy [[a]])` and `CHECK: _0 = opaque(move _3)`

Reply:
Addressed in https://github.com/rust-lang/rust/pull/135099/commits/6eab7c7e63fa6853413fa2ab483ecc75687b624b